libs/repositories/src/SimulationRepository/tenderlyTypes.ts

```diff
@@ -0,0 +1,574 @@
+export interface TenderlyBundleSimulationResponse {
+  simulation_results: TenderlySimulation[];
+}
+
+// types were found in Uniswap repository
+// https://github.com/Uniswap/governance-seatbelt/blob/e2c6a0b11d1660f3bd934dab0d9df3ca6f90a1a0/types.d.ts#L123
```

Comment:
I think we could but I would add it as a TODO to avoid blocking the PR merge. Btw, I tried to generate these types but I couldn't succeed, the gen was falling (https://docs.tenderly.co/api/openapi.json)